### ISSUE TITLE
Accessibility issue - lack of alt text and unlabeled buttons

### ISSUE THREAD

**Issue by esethna** [mattermost/mattermost-server#8455]
If you're interested please comment here and come [join our "Contributors" community channel](https://pre-release.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://pre-release.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://docs.mattermost.com/guides/developer.html), specifically for [machine setup](https://docs.mattermost.com/developer/developer-setup.html) and for [developer workflow](https://docs.mattermost.com/developer/developer-flow.html).

----

**Notes**: [Jira ticket](https://mattermost.atlassian.net/browse/MM-9820)

Original GitHub issue: https://github.com/mattermost/mattermost-server/issues/8446

I'm logging some accessibility issues in Mattermost that are effecting the usability for a blind user.

#### Summary
Lack of alt text and unlabelled buttons

#### Steps to reproduce
(Mattermost 4.4.2 using JAWS screen reader software)

#### Expected behavior
Everything should be labelled so that it gives alt text to the screen reader

#### Observed behavior
For example, the file upload/attach file button doesn't have alt text

**Comment by JayaKrishnaNamburu** [comment]
I will give my contribution for this

**Comment by sudheerDev** [comment]
Thanks, @JayaKrishnaNamburu. Let us know if you have any questions.

**Comment by jasonblais** [comment]
@sudheerDev Should I keep this one open for now?

**Comment by sudheerDev** [comment]
@jasonblais Yes, we need to update ticket with a list of elements that needs labels for accessibility. 
I can update with the list in a day or two.

Contributor says he can work on the elements in another PR. Lets wait for the other PR and then close this.

**Comment by sudheerDev** [comment]
@jasonblais Added few elements to the ticket above. We can close this once we have those elements are fixed as well.

Hey, @JayaKrishnaNamburu Let us know if you are interested in working on it.

**Comment by JayaKrishnaNamburu** [comment]
I will check the updates and start working on that as well and will give a PR shortly. 👍

**Comment by JayaKrishnaNamburu** [comment]
Hi @sudheerDev, @anushreesubramani started working on this and will refer her PR here.

**Comment by sudheerDev** [comment]
Sure sounds good! @JayaKrishnaNamburu. 
@anushreesubramani Thanks. Don't hesitate to reach out to us if you need any help!

**Comment by hanzei** [comment]
Reorging as https://github.com/mattermost/mattermost-server/issues/15931
